### PR TITLE
Expand nim support

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

**Comment by mawww** [comment]
Hello,

as this adds more than a few lines of code, and I dont think I have one from you already, can you add a copyright waiver commit as well (see the CONTRIBUTING file)

**Comment by SolitudeSF** [comment]
done

**Comment by mawww** [comment]
Thank you

File: rc/extra/nim.kak [hunk 1]
```diff
@@ -53,9 +53,37 @@ add-highlighter shared/nim/code regex %{'[^'\n]'} 0:string
...
+}
+
 # Initialization
 # ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
 
 hook -group nim-highlight global WinSetOption filetype=nim %{ add-highlighter window ref nim }
 
+hook global WinSetOption filetype=nim %{
+    set buffer tabstop 2
+    set buffer indentwidth 2
```
Comment (lenormf) [inline rc/extra/nim.kak]: Please remove those two lines, users are free to chose whatever indentation they want to edit nim code.

Comment (SolitudeSF) [reply rc/extra/nim.kak]: yeah, looking at other langs, they dont have this set.
its just i was looking at pony's script and it had indentation settings.
removed.

Comment (lenormf) [reply rc/extra/nim.kak]: Alright, they're gone now #1764.

**Pull request merged**